### ISSUE TITLE
wrong/missing alt text for the profile picture on the user profile page

### ISSUE THREAD

**Issue by marinafink** [marinafink/noshcuisine#8]
**Description:**
when logged in and going to the user profile, the profile picture has a wrong alt text, there is a space missing (example: profilepicture ofRosa). If the user does not have a profile picture, the placeholder picture that is displayed instead of the profile picture has no alt text at all.

**Steps to reproduce:**
1. Visit https://mars.iuk.hdm-stuttgart.de/~mf177/index.php (username: mf177, password: M5fFm67#ar)
2. Log in with the user data of a user who has a profile picture (user data for testing: username: Rosa, password: rosa123)
3. Go to the user profile
4. open the developer tools
5. use the inspector to see the HTML code of the profile picture
6. look at this tag: img class="rund_xl" src="files/WSR5K7FRFTEGUI5.jpg" alt="Profilepicture ofRosa"

also try with a user that has no profile picture:
1. Visit https://mars.iuk.hdm-stuttgart.de/~mf177/index.php (username: mf177, password: M5fFm67#ar)
2. Log in with the user data of a user who has no profile picture (user data for testing: username: Jannis K, password: hello)
3. Go to the user profile
4. open the developer tools
5. use the inspector to see the HTML code of the profile picture
6. look at this tag: img src="pic_collection/user_nopicture.png"

**Expected Result:**
For user with a profile picture:
The HTML img tag should have an alt text that says "Profilepicture of Username".

For user without profile picture:
The HTML img tag should have an alt text that says "No Profilepicture".

**Actual Result:**
For user with a profile picture:
The HTML img tag has an alt text, but it misses a space and therefore it says "Profilepicture ofUsername"

![Image](https://user-images.githubusercontent.com/86669759/225812411-c814c266-f5a1-4055-8f0d-90919b6aef79.png)

For user without profile picture:
The HTML img tag has no alt text at all.

![Image](https://user-images.githubusercontent.com/86669759/225812618-8f1a9f1a-23c6-4a90-a259-4ec043666217.png)

**Configuration:**

![](https://user-images.githubusercontent.com/86669759/225128352-92ecf32d-414f-4106-88fc-d5bf3547206c.png)

**Comment by marinafink** [comment]
The alt text for the profilepicture has been adjusted.
An alt text for "no profilepicture" has been added.
